#31/sample.rb

```diff
@@ -0,0 +1,9 @@
+class Author < ApplicationRecord
+  has_many :books
+end
+
+class Book < ApplicationRecord
+end
```

Comment:
belongs_toの定義追加をお願いします！

```suggestion
class Book < ApplicationRecord
  belongs_to :author
end
```